.ocamlformat-ignore

```diff
@@ -48,6 +47,8 @@ plugins/setoid_ring/*
 plugins/ssr/*
 plugins/ssrmatching/*
 plugins/syntax/*
-# Enabled: micromega
+# Enabled: micromega, funind
+# plugins/funind/*
 # plugins/micromega/*
 plugins/micromega/micromega.ml
+
```

Comment:
This extra new line is making the linter fail.

Reply:
Linter is failing due to non-matching ocamlformat versions, #11778 should address this.

Reply:
Could we switch to a positive listing of files instead of a negative one?